org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/info/InfoWindow.java

```diff
@@ -1051,10 +1055,44 @@ protected ColumnInfo createLookupColumnInfo(TableInfo[] tableInfos,
 			colSQL += " AS " + infoColumn.getColumnName();
         editorMap.put(colSQL, editor);
         Class<?> colClass = columnName.endsWith("_ID") || columnName.equals("CreatedBy") || columnName.equals("UpdatedBy") ? KeyNamePair.class : String.class;
-		ColumnInfo columnInfo = new ColumnInfo(infoColumn.getNameTrl(), colSQL, colClass, (String)null, infoColumn.isReadOnly() || haveNotProcess);
+        ColumnInfo columnInfo = new ColumnInfo(infoColumn.getNameTrl(), colSQL, colClass, (String)null, infoColumn.isReadOnly() || haveNotProcess, displayColumn);
 		return columnInfo;
 	}
 
+	/**
+	 * Check and parse the correct aliases of the display SQL
+	 * @param displayColumn
+	 * @return parsed displayColumn
+	 */
+	private String parseAliases(String displayColumn, String tableName) {
+		if(Util.isEmpty(displayColumn))
+			return "";
+		String alias = getAlias(tableName);
+		if(!displayColumn.contains(".") && !displayColumn.startsWith("NVL")) {
```

Comment:
That could be the common case in the core dictionary but I don't think there's any hard rule set on the display column sql here. What would happens here if the parse fail ? If it will just error out, I don't think this is acceptable.

Reply:
Instead of hard coded function name, I think it is better to look for the pattern of {function name}({column name}).

Reply:
Replaced with the right regex